rele/client.py

```diff
@@ -57,20 +63,31 @@ def create_subscription(self, subscription, topic):
         :param subscription: str Subscription name
         :param topic: str Topic name to subscribe
         """
-        subscription_path = self._client.subscription_path(
+        subscription_path = self._subscriber_client.subscription_path(
             self._gc_project_id, subscription
         )
-        topic_path = self._client.topic_path(self._gc_project_id, topic)
+        topic_path = self._subscriber_client.topic_path(self._gc_project_id, topic)
 
         with suppress(exceptions.AlreadyExists):
             try:
-                self._client.create_subscription(
-                    name=subscription_path,
-                    topic=topic_path,
-                    ack_deadline_seconds=self._ack_deadline,
-                )
+                self._create_subscription(subscription_path, topic_path)
             except exceptions.NotFound:
-                logger.error("Cannot subscribe to a topic that does not exist.")
+                logger.info(
+                    "Cannot subscribe to a topic that does not exist."
+                    "Creating topic..."
+                )
```

Comment:
Could you please include the name of the failing topic in the logs? That is precious information when you find these messages in Sentry.

Reply:
Done!